plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.xml

```diff
@@ -12,7 +12,6 @@
           <and>
             <requires facet="jst.web" version="2.5"/>
             <requires facet="java" version="1.7"/>
-            <conflicts facet="com.google.cloud.tools.eclipse.appengine.facets.flex"/>
```

Comment:
Can we comment this section out or add a note to remind us to add this conflict statement back when we reenable the flex facet